spaces/example/index.json

```
@@ -1,15 +1,16 @@
 {
-  "key": "example",
-  "name": "Example",
+  "key": "Ourbloc",
```

The key of your space must also be the folder name and skin class name.
```suggestion
  "key": "ourbloc",
```
You edited the `/spaces/example` space. You should create another directory `/spaces/ourbloc` for your space.